src/custom/components/AffiliateStatusCheck/index.tsx

```diff
@@ -97,6 +103,17 @@ export default function AffiliateStatusCheck() {
       return
     }
 
+    const hasJustTraded =
+      allRecentActivity
+        .filter(isConfirmed)
```

Comment:
I don't think that will apply because isConfirmed doesn't necessarily means it traded

```
  data.status === OrderStatus.FULFILLED || data.status === OrderStatus.EXPIRED || data.status === OrderStatus.CANCELLED
```

You are looking for only FULFILLED orders.

Unless we want the banner to disappear once the order is placed, then if there's any order for this account and chain you return true